src/utils.js

```diff
@@ -46,15 +46,23 @@ const writeFile = ({ filePath, content }) => new Promise((resolve, reject) => {
  * Sets up a basic HTML page to apply the content to
  * @return {xmlbuilder}
  */
-const createHtmlBase = ({ pageTitle, stylesheet }) => xmlbuilder.create({
-	html: {
-		head: {
-			meta: { '@charset': 'utf-8' },
-			title: { '#text': pageTitle },
-			style: { '@type': 'text/css', '#text': stylesheet },
+const createHtmlBase = ({ pageTitle, stylesheet, useCssFile }) => {
+	const htmlBase = {
+		html: {
+			head: {
+				meta: { '@charset': 'utf-8' },
+				title: { '#text': pageTitle },
+				style: { '@type': 'text/css', '#text': stylesheet },
+			},
 		},
-	},
-});
+	};
+
+	if (useCssFile) {
+		delete htmlBase.html.head.style;
```

Comment:
Instead of deleting the style, rather do a check whether to add the style-tag or the link tag (which means removing the style from the original object)